src/sonic-build-hooks/scripts/pre_run_buildinfo

```diff
@@ -1,7 +1,11 @@
 #!/bin/bash
 
+IMAGENAME=$1
+
 . /usr/local/share/buildinfo/scripts/buildinfo_base.sh
 
+#set -x 
```

Comment:
Could you please remove the unuse lines? Thanks.
Line 7, 18, 20.

```
#set -x
#set +x
```

Reply:
done